src/pu-main.c

```diff
@@ -220,13 +363,30 @@ static GOptionEntry option_entries_version[] = {
     { NULL }
 };
 
+static GOptionEntry option_entries_validate[] = {
+    { "skip-checksums", 's', G_OPTION_FLAG_NONE, G_OPTION_ARG_NONE,
+        &arg_validate_skip_checksums, "Skip checksum verification for all input files", NULL },
+    { "size", 0, G_OPTION_FLAG_NONE, G_OPTION_ARG_STRING,
+        &arg_validate_file_size, "Size of loop device. Default is 1G", "SIZE" },
```

Comment:
Again, a user may not be familiar with loop devices, so this description is not easy to understand without looking at the inner workings of partup.